utils/formatErrorMessage.test.js

```diff
@@ -0,0 +1,68 @@
+import test from 'node:test'
+import assert from 'node:assert/strict'
+
+import { formatErrorMessage } from './formatErrorMessage.js'
+
+test('formatErrorMessage should strip colors', async (_t) => {
+  try {
+    throw new Error('\u001B[0m\u001B[4m\u001B[42m\u001B[31mfoo\u001B[39m\u001B[49m\u001B[24mfoo\u001B[0m')
+  } catch (err) {
+    assert(formatErrorMessage(err) === 'foofoo')
+  }
+})
+
+test('formatErrorMessage should truncate length', async (_t) => {
+  try {
+    throw new Error('a'.repeat(200))
+  } catch (err) {
+    assert(formatErrorMessage(err) === 'a'.repeat(100) + '...')
+  }
+})
+
+test('formatErrorMessage should strip block formatting', async (_t) => {
+  const actualErrorMessage = `
```

Comment:
Up to you: You could move this into a text file under `/assets/fixtures` to (marginally) improve readability / maintainability.

Reply:
👍 done!